### PR TITLE
Redesign homepage

### PR DESCRIPTION
**Pull request by HenryHins**

### CODE REVIEW

**Comment by wparad** [comment]
I'll be able to validate this when I get back to a computer ~1week.

**Comment by HenryHins** [comment]
No worries

File: content/login/login.html [hunk 1]
```diff
@@ -54,33 +54,176 @@
     </div>
```
Comment (wparad) [inline content/login/login.html:54]: Almost everything looks good.  I like all the wording we have.  There is some image strangeness that I think we should clean up.  Take a look a screenshot, what can we do about that?

2017-06-28-215820_1523x772_scrot

**Review by wparad — CHANGES_REQUESTED** [review]

File: content/login/login.html (hunk 1 above)
Comment (HenryHins) [reply content/login/login.html:54]: Whoa are you using a 4K monitor or anything? What browser are you using? This doesn't look good. It looks like it's a monitor that has a very high resolution and a very low DPI?

**Comment by wparad** [comment]
I saw a little bit of something strange and noticed it was different with size of the screen, so I zoomed all the way to see the extent of the issue.

**Review by wparad — APPROVED** [review]

**Pull request merged**